pypeit/core/coadd.py

```diff
@@ -476,7 +478,7 @@ def median_filt_spec(flux, ivar, mask, med_width):
 def solve_poly_ratio(wave, flux, ivar, flux_ref, ivar_ref, norder, mask = None, mask_ref = None,
```

Comment:
Please remove use_huber option unless it is really needed.

Reply:
rm'd